WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -613,6 +617,24 @@ private extension DefaultProductFormTableViewModel {
                                                         isActionable: isActionable)
     }
 
+    func subscriptionFreeTrialRow(product: ProductFormDataModel, isEditable: Bool) -> ProductFormSection.SettingsRow.ViewModel {
+        let icon = UIImage.hourglass
+        let title = Localization.subscriptionFreeTrialTitle
+
+        let details: String = {
+            guard let subscription = product.subscription else {
+                return ""
+            }
+
+            return Localization.subscriptionFreeTrialDescription(trialLength: subscription.trialLength,
+                                                                 trialPeriod: subscription.trialPeriod)
```

Comment:
👀 Should we still display "No trial period" when we cannot find subscription? This is an edge case but we can make the logic simpler:

```suggestion
            let trialLength = product.subscription?.trialLength ?? ""
            let trialPeriod = product.subscriptions?.trialPeriod ?? ""

            return Localization.subscriptionFreeTrialDescription(trialLength: trialLength,  trialPeriod: trialPeriod)
```

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/11173/commits/6d9d13060e986ac683fc0c9ea2d86ded94d99953